<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWebView" name="qDefWeb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>741</width>
      <height>331</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>412</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="qGoogle">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Google</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qUrban">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Urban</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qLongman">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Longman</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qCollins">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Collins</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>741</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="QInputWord"/>
     </item>
     <item>
      <widget class="QPushButton" name="qPron">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Pronunciation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <signal>onPressEnterKey()</signal>
  <slot>showDefinitionWebpage()</slot>
 </slots>
</ui>
